### DISCUSSION TITLE
Dashboards show in dropdown V9.0

### DISCUSSION THREAD

**Discussion by JohanScheepers** [grafana/grafana#51075] [General]
Will it be possible to show all dashboards in the drop down menu?

![dropdown](https://user-images.githubusercontent.com/30056110/174478113-d9b8a0ee-2831-4ea3-b990-f4154a987cdd.PNG)

As nor you need to go dropdown -> Brows -> then select the dashboard.

![browse](https://user-images.githubusercontent.com/30056110/174478121-15a8492e-b4f3-4e60-9483-f5be4e730995.PNG)

**Comment by torkelo** [comment]
no but you can add starred dashboards to the nav bar 
https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-0/#starred-dashboards-in-the-navigation-bar

**Reply by JohanScheepers** [reply]
Thank you for the feedback.

Read the mentioned document and looked in the documentation.

Form the documentation I can see it is 4
![image](https://user-images.githubusercontent.com/30056110/174498908-7c6fb6da-9758-4a7c-bb12-a778600c9c20.png)

On my dashboard
![image](https://user-images.githubusercontent.com/30056110/174498948-2117b669-61f7-42ee-8dcb-50b72cf9aab3.png)

What am I missing here, do I need to enable the "Star dashboard" some where? Cant find anything in the grafana.ini

Running Open Source v9.0.0 on a Pi3B+ with the latest updates and upgrades.

**Reply by torkelo** [reply]
The link i posted mentions it's behind a feature flag

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
